OutOfSchool/OutOfSchool.WebApi/Controllers/V1/AdminController.cs

```diff
@@ -9,36 +12,181 @@ namespace OutOfSchool.WebApi.Controllers.V1;
 [Route("api/v{version:apiVersion}/[controller]/[action]")]
 public class AdminController : ControllerBase
 {
+    private readonly IStringLocalizer<SharedResource> localizer;
     private readonly IUserService userService;
+
+    private readonly ILogger<DirectionService> loggerDirection;
+    private readonly ILogger<MinistryAdminService> loggerMinistryAdmin;
+    private readonly ILogger<ProviderService> loggerProvider;
+
     private readonly IMapper mapper;
+    private readonly ICurrentUserService currentUserService;
+    private readonly ISensitiveMinistryAdminService ministryAdminService;
+    private readonly ICacheService cache;
+    private readonly ISensitiveDirectionService directionService;
+    private readonly ISensitiveProviderService providerService;
+    private readonly ISensitiveApplicationService applicationService;
 
-    public AdminController(IUserService userService, IMapper mapper)
+    public AdminController(
+        ILogger<DirectionService> loggerDirection,
+        ILogger<MinistryAdminService> loggerMinistryAdmin,
+        ILogger<ProviderService> loggerProvider,
```

Comment:
Only one `ILogger<AdminController> logger` is necessary. Please use it instead of separate "service" loggers.